scripts/item-descriptions.lua

```diff
@@ -0,0 +1,75 @@
+-- Holds custom descriptions for view-item-info
+-- By PeridexisErrant
+
+print("This is a content library; calling it does nothing.")
```

Comment:
The idea is that there are two ways for mods to override stuff on a per-save basis:  they can add or replace individual descriptions with `more-item-descriptions.lua`, or discard the whole thing and start over with `item-descriptions.lua`.  That may well be overcomplicated.

Re: implementation detail - reworking this is how I learnt lua, so I have no idea what specific structure you're suggesting but it's probably a good idea.